### DISCUSSION TITLE
OrangePi R1 Network setup

### DISCUSSION THREAD

**Discussion by hw-schrauber** [johang/sd-card-images#116] [Q&A]
Hi together,

I have been desperately trying for a few days how to activate my WLAN on the OrangePi R1. I downloaded the latest Debian Bookworm for the R1 here, copied it into the image with the boot, started the system on the R1 and also used parted to adapt the root partition to the size of the SD card. Of course, the whole thing was done via the ssh-terminal via LAN. Now I want to activate the WLAN and fail miserably. With "ifconfig" I get the following information:

end0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST> mtu 1500
        inet 192.168.60.221 netmask 255.255.255.0 broadcast 192.168.60.255
        inet6 fe80::42:dcff:fef6:fe18 prefixlen 64 scopeid 0x20<link>
        ether 02:42:dc:f6:fe:18 txqueuelen 1000 (Ethernet)
        RX packets 32774 bytes 3402391 (3.2 MiB)
        RX errors 0 dropped 20714 overruns 0 frame 0
        TX packets 1699 bytes 239654 (234.0 KiB)
        TX errors 0 dropped 0 overruns 0 carrier 0 collisions 0
        device interrupt 153

enxc0742bffb688: flags=4099<UP,BROADCAST,MULTICAST> mtu 1500
        ether c0:74:2b:ff:b6:88 txqueuelen 1000 (Ethernet)
        RX packets 0 bytes 0 (0.0 B)
        RX errors 0 dropped 0 overruns 0 frame 0
        TX packets 0 bytes 0 (0.0 B)
        TX errors 0 dropped 0 overruns 0 carrier 0 collisions 0

lo: flags=73<UP,LOOPBACK,RUNNING> mtu 65536
        inet 127.0.0.1 netmask 255.0.0.0
        inet6 ::1 prefixlen 128 scopeid 0x10<host>
        loop txqueuelen 1000 (Local Loopback)
        RX packets 80 bytes 6480 (6.3 KiB)
        RX errors 0 dropped 0 overruns 0 frame 0
        TX packets 80 bytes 6480 (6.3 KiB)
        TX errors 0 dropped 0 overruns 0 carrier 0 collisions 0

If I see this correctly, then the WLAN module is probably missing here. But somehow I can't get it to start. The vast majority of commands are acknowledged with “no command”. How can I get further? Why are the drivers for LAN and WLAN not already integrated in the image, if it is specially made for the board?

Can someone please help me!

**Comment by johang** [comment]
These images are very minimal on purpose. You can install more tools with apt, like in any Debian system. The WLAN module on Orange Pi R1 (rtl8189ftv) is not supported by mainline Linux or Debian, but I know there are drivers for rtl8189ftv out there on the internet that maybe and build yourself.
